distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,214 @@ public void testBasicRestAccess() throws Exception {
+
+        //try basic auth over http
+        try {
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url.replace("https://", "http://"));
+        } finally {
+            configureRestForGuest(SDK_SOAP_CONTEXT);
+            getSecurityPolicy().waitForGuestAuthReady(url);
+        }
+    }
+
+    @Test
+    public void testTLSv11() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.1");
+        sslContext.init(null, null, null);
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        checkAuthenticationRequest(config, url);
+    }
+
+    @Test
+    public void testTLSv12() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+        sslContext.init(null, null, null);
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        checkAuthenticationRequest(config, url);
+    }
+
+    @Test
+    public void testTLSv1() throws Exception {
+        String httpsProtocols = System.getProperties()
+                .getProperty("https.protocols");
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        try {
+            System.setProperty("https.protocols", "TLSv1");
+
+            SSLContext sslContext = SSLContext.getInstance("TLSv1");
+            sslContext.init(null, null, null);
+            SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+            RestAssuredConfig config = new RestAssuredConfig();
+            config.sslConfig(SSLConfig.sslConfig()
+                    .sslSocketFactory(sslSocketFactory));
+
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().config(config)
+                    .auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url)
+                    .then()
+                    .assertThat()
+                    .statusCode(401);
+        } finally {
+            System.setProperty("https.protocols", httpsProtocols);
+            configureRestForGuest(SDK_SOAP_CONTEXT);
+            getSecurityPolicy().waitForGuestAuthReady(url);
+        }
+    }
+
+    @Test
+    public void testAllowedCipherSuites() throws Exception {
```

Comment:
This test only verifies that at least one cipher suite was accepted.  It does not verify that they are all allowed. 
 The test will need to loop over each supported cipher and test it by itself to verify that all of these ciphers are supported.